src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,9 @@
-export const UserInfo = () => {};
+import { UserInfoProps } from '../../types/UserInfo';
+
+export const UserInfo: React.FC<UserInfoProps> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
+      {user?.name}
```

Comment:
Also, use a destruction for `user`
```suggestion
    <a className="UserInfo" href={`mailto:${user.email}`}>
      {user.name}
```